src/screens/users/list/Users.tsx

```diff
@@ -46,9 +49,20 @@ export default class Users extends BaseAutoRefreshScreen<Props, State> {
     this.setRefreshPeriodMillis(Constants.AUTO_REFRESH_LONG_PERIOD_MILLIS);
   }
 
+  public async componentDidMount(): Promise<void> {
+    this.userIDs = Utils.getParamFromNavigation(this.props.route, 'userIDs', null) as string;
+    this.carFullName = Utils.getParamFromNavigation(this.props.route, 'carFullName', null) as string;
+    this.carLicensePlate = Utils.getParamFromNavigation(this.props.route, 'carLicensePlate', null) as string;
+    await super.componentDidMount();
+  }
+
   public async getUsers(searchText: string, skip: number, limit: number): Promise<DataResult<User>> {
     try {
-      const users = await this.centralServerProvider.getUsers({ Search: searchText }, { skip, limit });
+      const params = {
+        Search: searchText,
+        UserID: this.userIDs
+      };
+      const users = await this.centralServerProvider.getUsers(params, { skip, limit });
```

Comment:
Rename transactionsNbrRecordsOnly to usersNbrRecordsOnly below + pass the same filter to call the API

Reply:
@AlixH Don't mark my comments solved, rule 44 violation ;-)